### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by wiistriker** [sematext/solr-autocomplete#8]
I just run code and working with demo phrases on solr 7.7.2. Everything is working okay, thank you for your code.

I have questions:

1) What is a current project status? Do you use it in production?

2) I am not very familiar with java, but it seems like this project is just a thin layer for other solr features? Can i reach same behaviour with solr without this plugin?

**Comment by otisg** [comment]
This project is not really maintained any more.

**Comment by wiistriker** [comment]
@otisg yes i see it by commit history. But i wonder why it not maintained? Do you use something else for autocomplete feature?
